packages/coinstac-ui/app/render/components/dashboard/dashboard.jsx

```diff
@@ -165,6 +165,13 @@ class Dashboard extends Component {
     });
 
     this.unsubscribeToUserMetadata = this.props.subscribeToUserMetaData(user.id);
+    
+    ipcRenderer.on('docker-error', (event, arg) => {
```

Comment:
You'll also see there are ipc `unsubscribes` for the event handlers, you'll want one for this handler too